pkg/util/backoff/backoff.go

```diff
@@ -80,19 +92,34 @@ func (b *Policy) GetBackoffDuration(numErrors int) time.Duration {
 }
 
 // IncError increments the error counter up to MaxErrors
-func (b *Policy) IncError(numErrors int) int {
+func (e *ExpBackoffPolicy) IncError(numErrors int) int {
 	numErrors++
-	if numErrors > b.MaxErrors {
-		return b.MaxErrors
+	if numErrors > e.MaxErrors {
+		return e.MaxErrors
 	}
 	return numErrors
 }
 
 // DecError decrements the error counter down to zero at RecoveryInterval rate
-func (b *Policy) DecError(numErrors int) int {
-	numErrors -= b.RecoveryInterval
+func (e *ExpBackoffPolicy) DecError(numErrors int) int {
+	numErrors -= e.RecoveryInterval
 	if numErrors < 0 {
 		return 0
 	}
 	return numErrors
 }
+
+// GetBackoffDuration returns amount of time to sleep after numErrors error
+func (c *ConstantBackoffPolicy) GetBackoffDuration(numErrors int) time.Duration {
+	return c.backoffTime
+}
+
+// IncError is a no-op here
+func (c *ConstantBackoffPolicy) IncError(numErrors int) int {
+	return 0
+}
```

Comment:
If `IncError`'s description is `IncError increments the number of errors and returns the new value`, it seems weird to always return 0 for the constant backoff policy.

There might be a piece of code assuming that it returns the real number of errors and returning 0 would break that...

I believe it would be better to store the real number to return it here, but always return the `backoffTime` in `GetBackoffDuration`.